cloud/wfm/src/executor/ExecutorImpl.ts

```diff
@@ -0,0 +1,37 @@
+import * as Promise from 'bluebird';
+import {Process} from '../process';
+import {ProcessInstance} from '../process-instance';
+import {Task, TaskEventData, TaskStatus} from '../task';
+import {Executor} from './index';
+import {InstanceRepository} from './InstanceRepository';
+
+export {InstanceRepository};
```

Comment:
IMHO that's antipattern. File should export all elements that are inside.
Then index should export all stuff is needed.
You do not want to jump thru files to know that their implemented somewhere else.